Pipfile

```
@@ -12,7 +12,7 @@ boto3 = "*"
 elasticsearch = "==7.13.4"
 jsonref = "*"
 jsonpickle = "*"
-pymongo= "*"
+pymongo= "==4.2.0"
```

this should stay as a `*`... the applications using this package should be the ones to limit the version, except in extreme cases like `elasticsearch`

```suggestion
pymongo= "*"
```

---

No problem